helper-cli/src/main/kotlin/commands/SplitCurations.kt

```diff
@@ -58,17 +58,18 @@ internal class SplitCurations : CliktCommand(
         }
 
         val packageCurations = inputCurationsFile.readValue<List<PackageCuration>>()
+        val groupedCurations = packageCurations.groupBy { it.copy(id = it.id.copy(version = "<dummy>")).id }
```

Comment:
Consider grouping it by output file instead of by (dummy) ID.

Reply:
Thanks for the hint. That makes way more sense. Changed the implementation.